src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundLiteralWhen.java

```diff
@@ -0,0 +1,42 @@
+/*
+WhitespaceAround
+tokens = LITERAL_WHEN
+
+*/
+
+//non-compiled with javac: Compilable with Java21
+package com.puppycrawl.tools.checkstyle.checks.whitespace.whitespacearound;
+
+public class InputWhitespaceAroundLiteralWhen {
```

Comment:
Add some examples of a new line before/after `when`.

Reply:
done